### PR TITLE
Fix #213 missing entries | Default Lemma Analyses for Word Classes 

### LINKED ISSUE #213: Cree word forms not getting some or all analyses
Pursuant #100 and #209, now the following Cree word forms are not recognized, either at all, or only receiving one of multiple possible analyses (`-`: analysis apparently missing; `+`: analysis apparently correct):

- [ ] `maskos (-maskwa+N+A+Der/Dim+N+A+Sg)`
- [ ] `maskwak (-maskwa+N+A+Pl)`
- [ ] `nimaskom (-maskwa+N+A+Px1Sg+Sg)`
- [ ] `niskak (-niska+N+A+Pl; +nîskâw+V+II+Cnj+Prs+3Sg)`
- [x] `nitas (-mitâs+N+A+D+Px1Sg+Sg; -mitâs+N+I+D+Px1Sg+Sg; -nitâs+N+A+D+Px1Sg+Sg; -nitâs+N+I+D+Px1Sg+Sg)` (two results with _nitâs_ as lemma, but `Sg` feature not showing)
- [x] `mitas (-mitâs+N+A+D+PxX+Sg; -mitâs+N+I+D+PxX+Sg)` (two results with _mitâs_ as lemma, but `Sg` feature not showing)
- [x] `mitasak (-mitâs+N+A+D+PxX+Pl)`

Dependent nouns in their lemma form are recognized, but other inflected forms such as plurals and obviatives are not.

### PR DESCRIPTION
**Pull request by Madoshakalaka**
#213 is related to our code's ability to identify xml entries with their fst analyses.

We used to rely purely on `lemma-tags.tsv` to identify xml entries with their fst analyses. It was flawed as there are idiomatic counterexamples. And for pronouns, there is no easy rule.

The newer code relied on comparing `<l> <pos> lc` in xml to fst analyzer outputs. There are cases this approach fails. e.g. the code has no way to know which one of `niska+N+A+Sg` or `niska+N+A+Obv` is the lemma analysis for _niska_ as both analyses look the same and has the same word class.

This PR added `lemma-tags.tsv` back, but as a fallback this time. As it's useful in disambiguating these cases.

I'll upload built production database when this pr is approved. (There is no `models.py` change so just a database swap and `$ touch wsgi.py` will work.)

### CODE REVIEW

File: cypress/integration/recordings.spec.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
  */
 context('Recordings', function () {
   describe('On the search page', () => {
-    it('should display for words  on the search page', () => {
```
Comment (Madoshakalaka) [inline cypress/integration/recordings.spec.js:6]: nitpick, "on the search page" was already inside `describe`

File: CreeDictionary/DatabaseManager/xml_entry_lemma_finder.py [hunk 2]
```diff
@@ -3,20 +3,65 @@
...
 
 
+class DefaultLemmaPicker:
+    """
+    When two analyses have the same looking lemma and word class, this helps choose the preferred lemma
+
+    i.e. It helps solve this question: "Which one of maskwa+N+A+Sg and maskwa+N+A+Obv is the lemma?"
+    """
+
+    def __init__(self, language: Literal["crk", "gunaha"]):
```
Comment (eddieantonio) [inline CreeDictionary/DatabaseManager/xml_entry_lemma_finder.py:29]:
```suggestion
    def __init__(self, language: Literal["crk", "srs"]):
```

If we're using the ISO 639 code for Tsuut'ina.

| Language | ISO 639-3 | App Name |
| - | - | - |
| Plains Cree | crk | itwêwina |
| Tsuut'ina | srs | Gúnahà |

File: CreeDictionary/tests/DatabaseManager_tests/test_xml_importer_regressions.py [hunk 3]
```diff
@@ -62,3 +62,23 @@ def test_import_pipon_of_different_word_classes(shared_datadir):
...
+
+
+@pytest.mark.django_db
+def test_import_maskwa(shared_datadir):
+
+    # https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/issues/213
+    # searching with maskwak didn't yield an inflection of "maskwa" as "maskwa" wasn't disambiguated and was marked as_is
+    migrate_and_import(shared_datadir / "crkeng-maskwa")
+
+    assert not Wordform.objects.filter(text="maskwa", is_lemma=True).get().as_is
```
Comment (eddieantonio) [inline CreeDictionary/tests/DatabaseManager_tests/test_xml_importer_regressions.py:84]: 👍

File: cypress/integration/recordings.spec.js (hunk 1 above)
Comment (eddieantonio) [reply cypress/integration/recordings.spec.js:6]: Haha, good point!

File: CreeDictionary/DatabaseManager/xml_entry_lemma_finder.py (hunk 2 above)
Comment (Madoshakalaka) [reply CreeDictionary/DatabaseManager/xml_entry_lemma_finder.py:29]: yyeaeeah thanks

**Pull request merged**